config_defaults.yml

```diff
@@ -1,6 +1,6 @@
 ---
 common:
-  disable_legacy_facts: false
+  disable_legacy_facts:true
```

Comment:
perhaps put the space back in between `:` and `true`

Reply:
ah yes. fixed it.